'ghcautoconf.h' and 'ghcplatform.h' are not found.

`ghcautoconf.h` is not marked as a rts dependency:
```
Reading shake-build/cfg/system.config...
| Successfully generated 'rts/stage1/package-data.mk'.
/----------
| Running HsCpp (stage = Stage1, package = rts):
|     input: rts/package.conf.in
| => output: rts/stage1/package.conf.inplace
\----------

In file included from rts/package.conf.in:3:
includes/ghcconfig.h:4:10: fatal error: 'ghcautoconf.h' file not found
#include "ghcautoconf.h"
         ^
1 error generated.
shakeArgsWith   0.000s    0%                           
Function shake  0.020s   34%  ===============          
Database read   0.006s    9%  ====                     
With database   0.000s    0%                           
Running rules   0.032s   55%  =========================
Total           0.057s  100%                           
Error when running Shake build system:
* rts/stage1/build/libHSrts.a
* OracleQ (PackageDataKey ("rts/stage1/package-data.mk","rts_stage1_C_SRCS"))
* rts/stage1/package-data.mk
* rts/stage1/package.conf.inplace
user error (Development.Shake.cmd, system command failed
Command: /usr/bin/clang -E -undef -traditional -Wno-invalid-pp-token -Wno-unicode -Wno-trigraphs -P -Iincludes -Iincludes/dist-derivedconstants/header -Icompiler/stage1 -x c rts/package.conf.in -DTOP="/Users/angerman/Projects/Haskell/ghc" -DFFI_INCLUDE_DIR= -DFFI_LIB_DIR= -DFFI_LIB="Cffi"
Exit code: 1
Stderr:
In file included from rts/package.conf.in:3:
includes/ghcconfig.h:4:10: fatal error: 'ghcautoconf.h' file not found
#include "ghcautoconf.h"
         ^
1 error generated.
)
```

after running:
```
./shake-build/build.sh includes/ghcautoconf.h 
```
we find that `ghcplatform.h` is also missing:
```
Reading shake-build/cfg/system.config...
| Successfully generated 'rts/stage1/package-data.mk'.
/----------
| Running HsCpp (stage = Stage1, package = rts):
|     input: rts/package.conf.in
| => output: rts/stage1/package.conf.inplace
\----------

In file included from rts/package.conf.in:3:
includes/ghcconfig.h:5:10: fatal error: 'ghcplatform.h' file not found
#include "ghcplatform.h"
         ^
1 error generated.
shakeArgsWith   0.000s    0%                           
Function shake  0.018s   37%  ==================       
Database read   0.005s   10%  =====                    
With database   0.000s    0%                           
Running rules   0.025s   51%  =========================
Total           0.049s  100%                           
Error when running Shake build system:
* rts/stage1/build/libHSrts.a
* OracleQ (PackageDataKey ("rts/stage1/package-data.mk","rts_stage1_C_SRCS"))
* rts/stage1/package-data.mk
* rts/stage1/package.conf.inplace
user error (Development.Shake.cmd, system command failed
Command: /usr/bin/clang -E -undef -traditional -Wno-invalid-pp-token -Wno-unicode -Wno-trigraphs -P -Iincludes -Iincludes/dist-derivedconstants/header -Icompiler/stage1 -x c rts/package.conf.in -DTOP="/Users/angerman/Projects/Haskell/ghc" -DFFI_INCLUDE_DIR= -DFFI_LIB_DIR= -DFFI_LIB="Cffi"
Exit code: 1
Stderr:
In file included from rts/package.conf.in:3:
includes/ghcconfig.h:5:10: fatal error: 'ghcplatform.h' file not found
#include "ghcplatform.h"
         ^
1 error generated.
)
```

--- a/src/Rules/Data.hs
+++ b/src/Rules/Data.hs
@@ -99,7 +99,9 @@ buildPackageData rs target @ (PartialTarget stage pkg) = do
                     fullTarget target (GhcPkg stage) [rtsConf] []
 
             rtsConf %> \_ -> do
-                need [rtsConfIn]
+                need [rtsConfIn
+                     , "includes/ghcautoconf.h"
+                     , "includes/ghcplatform.h" ]
                 build $ fullTarget target HsCpp [rtsConfIn] [rtsConf]
                 old <- liftIO $ readFile rtsConf
                 let new = unlines . map (replace "\"\"" "")

--- a/src/Rules/Generate.hs
+++ b/src/Rules/Generate.hs
@@ -62,6 +62,9 @@ generatedDependencies stage pkg
                         , "includes/ghcplatform.h" ]
     | pkg == rts      = let buildPath = targetPath stage rts -/- "build"
                         in
+                        [ "includes/ghcversion.h" -- missing only in stage1. See #76
+                        , derivedConstantsPath -/- "DerivedConstants.h" ]
+                        ++
                         fmap (buildPath -/-) ["ffi.h", "ffitarget.h"]
     | otherwise = []
 

